docs/en/stack/ml-biases.asciidoc

```diff
@@ -0,0 +1,96 @@
+[[ml-biases]]
+=== Considering biases in {ml} processes
+
+{ml-cap} is meant to improve our everyday life in a wide palette of activities 
+across the globe. We use {ml} in science, healthcare, business, entertainment, 
+and many other fields. We make decisions based on {ml} results. It is, 
+therefore, crucial to keep in mind the possible biases that can corrupt results, 
+endanger fairness, and jeopardize the success of even the best-intended 
+initiative. Some of these biases can make their way into the {ml} results more 
+easily than we might think.
+
+Various possible biases can negatively influence the result of a {ml} analysis. 
+In this piece, we'll talk about statistical and social biases (without 
+addressing the numerous subtypes of them individually), and not about the 
+possible biases in the {ml} algorithms themselves. Statistical and social biases 
+can exist in the datasets used to train {ml} models or in the way the 
+predictions of these models are interpreted. These issues are part of a greater 
+question about the ethics behind technology and there's no one solution for 
+everyone. However, there are some basic practices you can use to reduce these 
+biases.
+
+
+[float]
+==== Biases in datasets
+
+Biases exist in the data that we use to train {ml} models. For example, our 
+choice of which data to collect or analyze reflect our ideas about what is 
+important or worthy of attention. It is also possible that someone else has 
+collected the data for you. In the cases when you rely on others to do the data
+collection, you need to be even more aware of the possible consequences and 
+understand how the data has been collected and selected. Cleaning up "messy 
+data" is also an operation when the dataset could be biased. Messiness couldn't 
+be only the result of human error in the design or the collection process, but 
+instead an immanent characteristic of the real world. The bias can be also the 
+result of manual labeling when the dataset has been annotated with the "ground 
+truth", which is another opportunity to unwittingly introduce preconceptions or 
+prejudice. The model will inherently learn the bias, therefore the analysis will 
+be also compromised, while technically the algorithm works exactly as intended. 
+
+It is impossible to avoid all the potential biases in the dataset. Eventually, 
+data is a representation of reality and, like any other type of representation, 
+it never perfectly captures all possible variables affecting a given phenomenon. 
+However, you can and should carefully scrutinize your dataset, ensure it's not 
+selected in a predetermined way and it matches the real world as close as 
+possible. Involve people with diverse backgrounds in the processes that require 
+humans, it is also a solution to minimize the chance of biases.
```

Comment:
I would rewrite this sentence "Involve people with diverse backgrounds in the processes that require 
humans, it is also a solution to minimize the chance of biases." to say
"Involve people with diverse backgrounds in the design, construction and evaluation of ML systems."